src/page-sentencing/PageSentencing.js

```diff
@@ -4,20 +4,44 @@ import useChartData from "../hooks/useChartData";
 import VizSentenceTypes from "../viz-sentence-types";
 import VizSentenceLengths from "../viz-sentence-lengths";
 
+import VizSentencePopulation from "../viz-sentence-population";
+
 export default function PageSentencing() {
   const { apiData, isLoading } = useChartData("us_nd/sentencing");
 
   if (isLoading) {
     return null;
   }
 
+  const sentenceTypeByDistrictByDemographics =
+    apiData.sentence_type_by_district_by_demographics;
+  const populatedDistricts = new Set(
+    sentenceTypeByDistrictByDemographics.map((record) => record.district)
+  );
+  const districtIsPopulated = (record) => populatedDistricts.has(record.id);
+
   const TITLE = "Sentencing";
   const DESCRIPTION = `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
     Vestibulum in finibus tellus, et ullamcorper augue. Quisque eleifend
     tortor vitae iaculis egestas. Donec dictum, nunc nec tincidunt cursus,
     ipsum dui gravida.`;
 
   const SECTIONS = [
+    {
+      title: "Who is being sentenced?",
+      description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      Vestibulum in finibus tellus, et ullamcorper augue. Quisque eleifend
+      tortor vitae iaculis egestas. Donec dictum, nunc nec tincidunt cursus,
+      ipsum dui gravida.`,
+      showLocationControl: true,
+      locationControlLabel: "Judicial District",
+      VizComponent: VizSentencePopulation,
+      vizData: {
+        populationDemographics: sentenceTypeByDistrictByDemographics,
+        locations: apiData.judicial_districts.filter(districtIsPopulated),
```

Comment:
The `apiData.judicial_districts` call returns a district called "[Other](https://github.com/Recidiviz/public-dashboard/blob/master/server/core/demo_data/judicial_districts.json#L9)" such that it is displayed in the location dropdown control.  However, the `apiData.sentence_type_by_district_by_demographics` doesn't return any records with a district mapped to "Other".  The effect is that when the "Other" option is selected from the location control, an error is thrown and the page blows up.  To get around this issue, I filter the `apiData.judicial_districts` down to only include districts that are found within the data returned by `apiData.sentence_type_by_district_by_demographics`.

Reply:
@macfarlandian I can imagine cases where this would _not_ be the ideal behavior. For example, in cases where some particular breakdown by time and demographic leads to a 0 for a particular district, would we want to display some nominal pointer to that district instead of just relying on its absence for communication?

Reply:
yeah, unfortunately rows that zero out are simply dropped from the data file, so we can't rely on file contents to be exhaustive; `judicial_districts` is the source of truth. If there is a missing district we should plug that gap with zeroes or "no data" states as required.